test/e2e/storage/testsuites/multivolume.go

```diff
@@ -179,6 +179,14 @@ func (t *multiVolumeTestSuite) DefineTests(driver TestDriver, pattern testpatter
 		if l.config.ClientNodeName != "" {
 			framework.Skipf("Driver %q requires to deploy on a specific node - skipping", l.driver.GetDriverInfo().Name)
 		}
+		// For multi-node tests there must be enough nodes with the same toopology to schedule the pods
+		nodeSelection := e2epod.NodeSelection{Name: l.config.ClientNodeName}
```

Comment:
I guess the line above means that ClientNodeName should be empty. So let's just not set anything here?

Reply:
Repeat throughout

Reply:
This is only re-doing what was already in the code before: check the next chunk -- I simply didn't want to change what was there already.